torchaudio/models/wav2vec2/model.py

```diff
@@ -29,29 +30,38 @@ def __init__(
         self.feature_extractor = feature_extractor
         self.encoder = encoder
 
+    @torch.jit.export
     def extract_features(
```

Comment:
As offline discussed, we are thinking if users will want to fine-tune the Wave2Vec model for their down-stream task. If they just want the embeddings of Wave2Vec, this should be fine, we just need to set the model to eval mode to avoid training it. However, if they want to re-train the weights of some layers, it's good to add a method to freeze the unused layers and make ``requires_grad`` be True for the used layers.